x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -259,9 +258,6 @@ export const SessionView = ({
     <div css={styles.sessionViewerComponent}>
       <EuiPanel hasShadow={false} borderRadius="none" className="sessionViewerToolbar">
         <EuiFlexGroup alignItems="center" gutterSize="s">
-          <EuiFlexItem grow={false}>
-            <EuiBetaBadge label={BETA} size="s" css={styles.betaBadge} />
```

Comment:
can you remove the betaBadge style in the styles file, as it's not used anymore.

Reply:
the translation is still used by the tty player beta badge though, so that can stay.